packages/common/utils/src/mapValues.ts

```diff
@@ -11,7 +13,10 @@
  * mapValues({ foo: 1, bar: 2 }, x => x * 2)
  * // => { foo: 2, bar: 4 }
  */
-export function mapValues<T, U>(value: T, mapper: (value: T[keyof T]) => U): { [K in keyof T]: U } {
+export function mapValues<T extends Record<string | number, T[ObjectKeys<T>]>, U>(
+  value: T,
+  mapper: (value: T[keyof T]) => U
+): { [K in keyof T]: U } {
```

Comment:
I think this isn't correct.

`extends` keyword doesn't mean 'strict equal' to passed type. You can just specify this generic should be passed as object, not the same type of `T`.

And `object.entries` accepts any object even if it has symbol keys. Symbol keys are only excluded from return value of `object.entries`.

So I think the definition of `mapValues` function should be:
```typescript
export function mapValues<T extends Record<PropertyKey, any>, U, O = Omit<T, symbol>>(
  value: T,
  mapper: (value: T[keyof O]) => U
): { [K in keyof O]: U };
```

P.S. `extends` constraint is just sub-type of the type passed into generic. So your generic accepts an object with symbol keys after all.

Reply:
I uderstand mapValues should also aceept symbol, but object entries should exclude symbol type.
Is it right? 

I include your opinion in below commit.
[fix: make mapValue get symbol type parameter](https://github.com/toss/slash/pull/60/commits/7b5f449cbea1334ff93733c49d834f4262d46aa7)

And I edit mapvValues JSDoc.
[docs: edit mapValues JSDoc](https://github.com/toss/slash/pull/60/commits/624b8c6d40e384d3a50fda0424743c17ad92a9e6)

I came into this PR to contribute your library but I learn many things about type design.
Thank you for your kind explain.

Reply:
OK. 
Thank you!!!